src/converters.js

```diff
@@ -885,6 +885,8 @@ function injectViewList( modelItem, injectedItem, mapper, removePosition ) {
 		}
 	}
 
+	insertPosition = positionAfterUiElements( insertPosition );
```

Comment:
It's not always a position at the end of `<li>`. Sometimes it is between two `<ul>`s (that's when `<ul>` is broken to make a place injected `<ul><li>...</li></ul>` structure). Sometimes weird cases happen.

I know it sounds easy: "just find correct `<li>` and insert the list at the end of it, DUH?", but writing those converters wouldn't take weeks if it was so.

For example I remember time when positions were created a lot "at the end of `<li>`" and it lead to weird problems, like, let's say you have a `<li>` with `<ul>` and then you try to inject view list item. But you have to inject it before that `<ul>` not after, because this is how they are ordered in model. You cannot set the insert position to the end, because something (`<ul>`) may already be there and that would be incorrect.

So, just believe me that the way it is is safe :)